packages/devextreme/js/__internal/grids/grid_core/columns_resizing_reordering/m_columns_resizing_reordering.ts

```diff
@@ -156,16 +164,28 @@ const SeparatorView = modules.View.inherit({
         return getWidth($element);
       }
     }
-  },
-});
+  }
+}
+
+class ColumnsSeparatorView extends SeparatorView {
+  private _positionChanged: any;
+
+  private _tablePositionController: any;
+
+  private _isTransparent?: boolean;
+
+  /// #DEBUG
+  private _testPosX?: number;
+
+  private _testCursorName?: string;
+  /// #ENDDEBUG
 
-const ColumnsSeparatorView = SeparatorView.inherit({
   _renderSeparator() {
-    this.callBase();
+    super._renderSeparator();
 
     const $element = this.element();
     $element.addClass(this.addWidgetPrefix(COLUMNS_SEPARATOR_CLASS));
-  },
+  }
 
   _subscribeToCallback() {
     const that = this;
```

Comment:
Let's replace ```that``` -> ```this``` if it's possible :)
(We can use the arrow function in the ```_positionChanged``` to prevent loosing context)